packages/seacas/libraries/ioss/src/Ioss_CoordinateFrame.C

```diff
@@ -23,4 +23,37 @@ namespace Ioss {
   const double *CoordinateFrame::origin() const { return &pointList_[0]; }
   const double *CoordinateFrame::axis_3_point() const { return &pointList_[3]; }
   const double *CoordinateFrame::plane_1_3_point() const { return &pointList_[6]; }
+
+  bool Ioss::CoordinateFrame::operator==(const Ioss::CoordinateFrame &rhs) const
+  {
+    if( this->id_ != rhs.id_ ) { 
```

Comment:
Sorry, all output should be via lib::fmt.  Also, why is `operator==` outputting anything anyway...?

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.